docs/api/source/index.rst

```diff
@@ -13,7 +13,24 @@ For Solid Servers that support notifications, they are sent when a resource is
 created, updated, deleted, or when a child resource is added to or removed from
 a container.
 
-**This library currently only supports notifications sent via WebSockets.**
+.. admonition:: Supported Protocols
+   :class: caution
+
+   This library currently only supports an `early version of WebSocket
+   notifications
+   <https://docs.inrupt.com/ess/1.1/services/service-websocket/>`__, which
+   formed the basis of the Solid Notification Protocol 1.0 draft specification.
+
+   The `Solid Notification Protocol 1.0
+   <https://solid.github.io/notifications/protocol>`__ is a draft specification
+   undergoing technical review as it matures into a formal recommendation. A
+   future release will support this draft specification once implementations
+   become available. 
+   
+   **Please note:** the `legacy (insecure) WebSocket
```

Comment:
So ... I would refrain from labeling this as "insecure" because then people will ask you to enumerate in all ways you mean by insecure, etc. 
But will let @nicolasmondada  weigh in.

Reply:
This wording largely came from @acoburn — perhaps he can give some light on this.

Reply:
Let's avoid using nouns on things we don't recommend support and be specific that "Inrupt" does not recommend it. Let's just say: "The legacy Websocket API, which is not recommended by Inrupt for secure messaging, is not supported  by this library

Reply:
The comment on security: https://github.com/solid/solid-spec/blob/master/api-websockets.md#status